### PR TITLE
[T2A2][T09-A3]Chew Wan Theng Ruth

### PR DESCRIPTION
**Pull request by chewwt**

### CODE REVIEW

**Comment by chewwt** [comment]
ready for review

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -783,11 +783,11 @@ private static void deletePersonFromAddressBook(int index) {
      * @return true if the given person was found and deleted in the model
      */
     private static boolean deletePersonFromAddressBook(String[] exactPerson) {
-        final boolean changed = ALL_PERSONS.remove(exactPerson);
-        if (changed) {
+        final boolean isChanged = ALL_PERSONS.remove(exactPerson);
```
Comment (mauris) [inline src/seedu/addressbook/AddressBook.java:794]: this makes the variable clearer! good change

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -203,13 +203,17 @@ public static void main(String[] args) {
...
         while (true) {
-            String userCommand = getUserInput();
-            echoUserCommand(userCommand);
-            String feedback = executeCommand(userCommand);
-            showResultToUser(feedback);
+            obtainAndExecuteCommand();
         }
     }
 
+	private static void obtainAndExecuteCommand() {
```
Comment (mauris) [inline src/seedu/addressbook/AddressBook.java:210]: nice attempt to SLAP the main method!

**Comment by mauris** [comment]
good job improving the code quality!

**Pull request closed without merging**